styles/pythonanywhere/catppuccin.user.css

```diff
@@ -0,0 +1,1330 @@
+    @overlay1: @catppuccin[@@lookup][@overlay1];
+    @overlay0: @catppuccin[@@lookup][@overlay0];
+    @surface2: @catppuccin[@@lookup][@surface2];
+    @surface1: @catppuccin[@@lookup][@surface1];
+    @surface0: @catppuccin[@@lookup][@surface0];
+    @base: @catppuccin[@@lookup][@base];
+    @mantle: @catppuccin[@@lookup][@mantle];
+    @crust: @catppuccin[@@lookup][@crust];
+    @accent-color: @catppuccin[@@lookup][@@accent];
+
+    --ctp-rosewater: @rosewater;
+    --ctp-flamingo: @flamingo;
+    --ctp-pink: @pink;
+    --ctp-mauve: @mauve;
+    --ctp-red: @red;
+    --ctp-maroon: @maroon;
+    --ctp-peach: @peach;
+    --ctp-yellow: @yellow;
+    --ctp-green: @green;
+    --ctp-teal: @teal;
+    --ctp-sky: @sky;
+    --ctp-sapphire: @sapphire;
+    --ctp-blue: @blue;
+    --ctp-lavender: @lavender;
+    --ctp-text: @text;
+    --ctp-subtext1: @subtext1;
+    --ctp-subtext0: @subtext0;
+    --ctp-overlay2: @overlay2;
+    --ctp-overlay1: @overlay1;
+    --ctp-overlay0: @overlay0;
+    --ctp-surface2: @surface2;
+    --ctp-surface1: @surface1;
+    --ctp-surface0: @surface0;
+    --ctp-base: @base;
+    --ctp-mantle: @mantle;
+    --ctp-crust: @crust;
+
+    color-scheme: if(@lookup = latte, light, dark);
+
+    ::selection {
+      background-color: fade(@accent-color, 30%) !important;
+    }
+
+    input,
+    textarea {
+      &::placeholder {
+        color: @subtext0 !important;
+      }
+    }
+
+    .ace_editor {
+      background-color: @crust;
+      color: @text;
+
+      .ace_gutter {
+        background: @mantle;
+        color: @subtext0;
+      }
+      .ace_print-margin {
+        background: @mantle;
+      }
+
+      .ace_marker-layer {
+        .ace_active-line {
+          background-color: fade(@yellow, 20%);
+        }
+        .highlight-line-error {
+          background-color: fade(@red, 20%);
+        }
+        .ace_bracket {
+          border-color: @overlay1;
+        }
+      }
+
+      .ace_cursor {
+        color: @text;
+      }
+      .ace_marker-layer .ace_selection {
+        background: fade(@accent-color, 30%);
+      }
+      .ace_fold {
+        background-color: @surface0;
+        border-color: @surface1;
+      }
+
+      .ace_constant.ace_language,
+      .ace_keyword,
+      .ace_meta,
+      .ace_variable.ace_language {
+        color: @mauve;
+      }
+      .ace_line .ace_identifier:not(:first-of-type) {
+        color: @blue;
+      }
+      .ace_paren {
+        color: @red;
+      }
+      .ace_constant.ace_numeric {
+        color: @peach;
+      }
+      .ace_entity.ace_other.ace_attribute-name,
+      .ace_support.ace_constant,
+      .ace_support.ace_function {
+        color: @teal;
+      }
+      .ace_entity.ace_name.ace_tag,
+      .ace_support.ace_class,
+      .ace_support.ace_type {
+        color: @blue;
+      }
+      .ace_storage {
+        color: @peach;
+      }
+      .ace_string {
+        color: @green;
+      }
+      .ace_comment {
+        color: @surface2;
+      }
```

Comment:
A bunch of these look wrong, make sure they match our other syntax highlighting ports.

Reply:
this code appears to be the exact same as [this userstyle](https://github.com/catppuccin/userstyles/blob/195d5b63822209bf024cf91f655c60823ee0b5cc/styles/trinket/catppuccin.user.css#L224)

Reply:
i checked one of the syntax highlighting ports and they appear to all match

Reply:
Comments aren't supposed to be surface2.

Reply:
then why does one of the syntax highlighting ports use ``surface2`` for its comments?
![image](https://github.com/user-attachments/assets/8f4ceea6-7e72-4263-94d4-d46138211a30)

Reply:
https://www.merriam-webster.com/dictionary/mistake

https://github.com/catppuccin/vscode/blob/2d7388ffe7321f588b2f54e2ffaab9fb7c9dde14/packages/catppuccin-vsc/src/theme/tokens/index.ts#L50-L57

Reply:
> https://www.merriam-webster.com/dictionary/mistake
you don't need to link to the dictionary for me to understand its a mistake lol